ecl/eclcmd/eclcmd_common.hpp

```diff
@@ -167,6 +167,7 @@ typedef IEclCommand *(*EclCommandFactory)(const char *cmdname);
 #define ECLOPT_MANIFEST_DASH "-manifest"
 #define ECLOPT_LEGACY "--legacy"
 #define ECLOPT_LEGACY_DASH "-legacy"
+#define ECLOPT_CHECKDIRTY "-checkDirty"
```

Comment:
Two dashes is generally preferred for long form eclcmd option names.  Legacy and manifest are weird cases where for some reason I support both forms.  Could support both for this too, but I think that was originally thinking people would be used to the eclcc version, but it usually isn't the case that eclcmd users also use eclcc directly.